x-pack/plugins/observability/public/components/shared/exploratory_view/series_editor/report_metric_options.test.tsx

```diff
@@ -54,4 +55,23 @@ describe('ReportMetricOptions', function () {
 
     expect(await screen.findByText('Page load time')).toBeInTheDocument();
   });
+
+  it('should include a tooltip for the report metric', async function () {
+    mockAppIndexPattern({ loading: false });
+    const { getByText } = render(
+      <ReportMetricOptions
+        seriesId={0}
+        seriesConfig={{ ...dataViewSeries }}
+        series={{ ...mockUxSeries }}
+      />
+    );
+
+    userEvent.hover(getByText('Page load time'));
+
+    // The tooltip from EUI takes 250ms to appear, so we must
+    // use waitFor to asynchronously poll for it.
+    await waitFor(() => {
+      expect(getByText('Report metric')).toBeInTheDocument();
+    });
```

Comment:
instead of wrapping getByText in waitFor, it's better to use findByText, which already does waitFor built in.

Reply:
Good shout!